src/components/ordersPage/content.js

```diff
@@ -4,21 +4,142 @@
 
 export default class Content {
     constructor() {
+        this.data = window.OrdersData;
+        this.orders = window.OrdersData.data;
     }
 
     init = () => {
+ 
     }
 
+
     render = () => {
+        this.addOrderFunc();
         var content = $('<div class="content-container"></div>');
-        $(content).append(this.createInformationBox());
+        $(content).append(this.allOrders());
+         window.lang.generateStrings(content);
         return content;
     }
 
-    createInformationBox = () => {
-        var content = $('<div class="info-page-container">' +
-                            '<p>Orders test</p>'+
-                        '</div>');
-        return content;
+//test function used for adding orders
+    addOrderFunc = () => {
+
+            this.data.addOrder(4, [{item: "beer", price: "42"}]);
+
+            this.data.addOrder(5, [{item: "drink", price: "89"}]);
+
+            this.data.addOrder(5, [{item: "beer", price: "39"},{item: "wine", price: "49"}]);
+
+
     }
+
+
+    allOrders = () => {
+        let all_orders = $('<div id="all-orders"></div>');
+
+        let orders = $('<div  id="orders"></div>');
+        let del_orders = $('<div  id="delivered-orders"></div>');
+        let header_order = $('<div class="header-order"><span data-textid="order-header-order"></span></div>');
+        let header_del = $('<div class="header-delivered"><span data-textid="order-header-delivered"></span></div>');
+        let cont_orders = $('<div  id="orders-container"></div>');
+        let cont_del_orders = $('<div id="delivered-orders-container" ></div>');
+
+        $(all_orders).append(orders);
+        $(all_orders).append(del_orders);
+        $(del_orders).append(header_del);
+        $(orders).append(header_order);
+        $(del_orders).append(cont_del_orders);
+        $(orders).append(cont_orders);
```

Comment:
If possible: seperate these so they are appended together with their specific category. They are created and appended every other, so first "order" then "del_order" then "order", would be easier if it was like:
```suggestion
        /* Order items */
        let orders = $('<div  id="orders"></div>');
        let header_order = $('<div class="header-order"><span data-textid="order-header-order"></span></div>');
        let cont_orders = $('<div  id="orders-container"></div>');
        
        $(orders).append(header_order);
        $(orders).append(cont_orders);
        
        /* Delivered order items */
        let del_orders = $('<div  id="delivered-orders"></div>');
        let header_del = $('<div class="header-delivered"><span data-textid="order-header-delivered"></span></div>');
        let cont_del_orders = $('<div id="delivered-orders-container" ></div>');
        
        $(del_orders).append(header_del);
        $(del_orders).append(cont_del_orders);
        
        $(all_orders).append(orders);
        $(all_orders).append(del_orders);
```